dev-docs/source/GettingStarted.rst

```diff
@@ -55,6 +55,19 @@ Unfortunately CMake can't find it out of the box and the following steps are req
 Linux
 -----
 
+The goal of this section is to (in broad terms) get to the state where one can build and run a subset of tests
+
+.. code-block:: sh
+
+   git clone git@github.com:mantidproject/mantid.git
+   cd mantid
+   mkdir build  # this makes an in-source build
+   cd build
+   cmake3 -GNinja ../  # wrap in scl enable on RHEL7
+   ninja all AlgorithmsTest  # ninja-build on RHEL7
+   ctest -R ^AlgorithmsTest --output-on-failure
```

Comment:
It is matching the test names. Each test is named `<package>Test_<testname>`. For example, to run the tests for `Rebin` algorithm, the command would be `ctest -R ^AlgorithmsTest_RebinTest$ --output-on-failure`. The anchors are because "rebin" is a somewhat common name. Try with and without and you'll see if it makes a difference.

Reply:
I see, learned something new today :smile: